README.rst

```diff
@@ -1,11 +1,11 @@
 Lookup Account Authentication Action Plugin
 =============================================
 
-.. image:: https://img.shields.io/badge/quality-demo-red
+.. image:: https://img.shields.io/badge/quality-production-green
    :target: https://curity.io/resources/code-examples/status/
    :alt: Quality
 
-.. image:: https://img.shields.io/badge/availability-source-blue
+.. image:: https://img.shields.io/badge/availability-bundled-green
```

Comment:
Is it too early to change these? It won't be bundled until 7.5. Perhaps we can add a "since 7.5" below?

Reply:
That's a good point. Yes, it won't be bundled until 7.5. Probably it is better to revert this change and only update the badges after we have 7.5. Will do that.

Reply:
I'll leave the `quality-production-green` and change back to `availability-source-blue`.